internal/gengapic/genrest.go

```diff
@@ -1135,7 +1135,11 @@ func (g *generator) unaryRESTCall(servName string, m *descriptorpb.MethodDescrip
 	p("  httpReq = httpReq.WithContext(ctx)")
 	p("  httpReq.Header = headers")
 	p("")
-	p("  buf, err := executeHTTPRequest(ctx, c.httpClient, httpReq, c.logger, %s, %q)", logBody, m.GetName())
+	if isHTTPBodyMessage {
+		p("  buf, httpRsp, err := executeHTTPRequestWithResponse(ctx, c.httpClient, httpReq, c.logger, %s, %q)", logBody, m.GetName())
```

Comment:
How does `httpRsp` get used in the client?

Reply:
I would direct you to the updated generated file: internal/gengapic/testdata/rest_HttpBodyRPC.want

Reply:
Thanks, when I look at the whole file I see the usage:

``
if headers := httpRsp.Header; len(headers["Content-Type"]) > 0 {
			resp.ContentType = headers["Content-Type"][0]
		}
```

But it doesn't show up in the diff. Odd that this usage existed before.